src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -0,0 +1,139 @@
+import type {OptionType} from './ToggleSettingsOptionRow';
+
+type WorkspaceWorkflowsPageProps = WithPolicyProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.WORKFLOWS>;
+
+function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    const items: OptionType[] = [
+        {
+            icon: Illustrations.ReceiptEnvelope,
+            title: translate('workflowsPage.delaySubmissionTitle'),
+            subtitle: translate('workflowsPage.delaySubmissionDescription'),
+            onToggle: (isEnabled: boolean) => {
+                Policy.setWorkspaceAutoReporting(route.params.policyID, isEnabled);
+            },
+            subMenuItems: (
+                <MenuItem
+                    title={translate('workflowsPage.submissionFrequency')}
+                    titleStyle={styles.workspaceWorkflowsSubMenuTitle}
+                    descriptionTextStyle={styles.workspaceWorkflowsSubMenuDescription}
+                    description={translate('workflowsPage.weeklyFrequency')}
+                    shouldShowRightIcon
+                    wrapperStyle={styles.workspaceWorkflowsSubMenuContainer}
+                    hoverAndPressStyle={[styles.mr0, styles.br2]}
+                />
+            ),
+            hasBeenToggled: policy?.harvesting?.enabled ?? false,
+        },
+        {
+            icon: Illustrations.Approval,
+            title: translate('workflowsPage.addApprovalsTitle'),
+            subtitle: translate('workflowsPage.addApprovalsDescription'),
+            onToggle: (isEnabled: boolean) => {
+                Policy.setWorkspaceApprovalMode(route.params.policyID, policy?.owner ?? '', isEnabled ? 'BASIC' : 'OPTIONAL');
+            },
+            subMenuItems: (
+                <MenuItem
+                    title={translate('workflowsPage.approver')}
+                    titleStyle={styles.workspaceWorkflowsSubMenuTitle}
+                    descriptionTextStyle={styles.workspaceWorkflowsSubMenuDescription}
+                    description={policy?.owner ?? ''}
+                    // onPress={() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_APPROVER.getRoute(route.params.policyID))}
+                    shouldShowRightIcon
+                    wrapperStyle={styles.workspaceWorkflowsSubMenuContainer}
+                    hoverAndPressStyle={[styles.mr0, styles.br2]}
+                />
+            ),
+            hasBeenToggled: policy?.isAutoApprovalEnabled ?? false,
+        },
+        {
+            icon: Illustrations.WalletAlt,
+            title: translate('workflowsPage.makeOrTrackPaymentsTitle'),
+            subtitle: translate('workflowsPage.makeOrTrackPaymentsDescription'),
+            onToggle: () => {
+                // TODO call API routes && set onyx optimistic data
```

Comment:
Can we just link to the issue that will implement this instead?

Reply:
Done